gcp_deepvariant_runner.py

```diff
@@ -322,6 +323,15 @@ def _get_gcs_relative_path(gcs_path):
   return urlparse.urlparse(gcs_path).path.strip('/')
 
 
+def _meets_gcp_label_restrictions(label):
+  """Does given string meet GCP label restrictions?"""
+  max_suffix_len = max(len(_MAKE_EXAMPLES_JOB_NAME),
+                       len(_CALL_VARIANTS_JOB_NAME),
+                       len(_POSTPROCESS_VARIANTS_JOB_NAME))
+  return (re.match(re.compile(r'^[a-z][a-z0-9_-]*$'), label) is not None and
```

Comment:
I'm not sure what regex syntax python supports but you should be able to inline the length check into the expression itself...

Reply:
Good idea, done!